modules/cluster-logging-upgrading-elasticsearch.adoc

```diff
@@ -1,81 +1,55 @@
 // Module file include in the following assemblies:
-//cluster-logging-upgrading.adoc
-:_content-type: PROCEDURE
-[id="cluster-logging-updating-logging-to_current_{context}"]
-= Updating Logging to the current version
-
-To update Logging to the current version, you change the subscriptions for the OpenShift Elasticsearch Operator and Red Hat OpenShift Logging Operator.
+// logging/cluster-logging-upgrading.adoc
 
-[IMPORTANT]
-====
-You must update the OpenShift Elasticsearch Operator _before_ you update the Red Hat OpenShift Logging Operator. You must also update _both_ Operators to the same version.
-====
+:_content-type: PROCEDURE
+[id="cluster-logging-upgrading-elasticsearch_{context}"]
+= Updating the OpenShift Elasticsearch Operator
 
+To update the OpenShift Elasticsearch Operator to the current version, you must modify the subscription.
 
-If you update the Operators in the wrong order, Kibana does not update and the Kibana custom resource (CR) is not created. To work around this problem, you delete the Red Hat OpenShift Logging Operator pod. When the Red Hat OpenShift Logging Operator pod redeploys, it creates the Kibana CR and Kibana becomes available again.
+include::snippets/logging-elastic-dep-snip.adoc[]
 
 .Prerequisites
 
-* The {product-title} version is 4.7 or later.
+* If you are using Elasticsearch as the default log store, and Kibana as the UI, update the OpenShift Elasticsearch Operator before you update the Cluster Logging Operator.
++
+[IMPORTANT]
+====
+If you update the Operators in the wrong order, Kibana does not update and the Kibana custom resource (CR) is not created. To fix this issue, delete the Red Hat OpenShift Logging Operator pod. When the Red Hat OpenShift Logging Operator pod redeploys, it creates the Kibana CR and Kibana becomes available again.
```

Comment:
@periklis @xperimental  is this accurate now that EO controls certificates.  My expectation would be:

* EO updates to latest
* EO reconciles existing objects
* EO is able to diff kibana objects which may include an image update
* New kibana deployment

I do not expect any of this is contingent upon the CLO doing anything with an existing Kibana resource.

Reply:
This was content that already existed in  the docs. Since the merge freeze is today and I am out on Monday and Tuesday, let's follow up additional changes here post-5.8 release in the interest of trying to get this merged ASAP?

Reply:
wfm